### PR TITLE
DBZ-8142 Fix conversion of `BigInteger` timestamp values

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-8142

### CODE REVIEW

**Comment by Naros** [comment]
The XStream ORA-01466 error, while fixed in DBZ-7072, isn't used during CI tests.
Enabled feature for CI in https://github.com/debezium/debezium/pull/5811

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/olr/OpenLogReplicatorValueConverter.java [hunk 1]
```diff
@@ -113,7 +114,7 @@ protected Object convertTimestampWithZone(Column column, Field fieldDefn, Object
 
     @Override
     protected Object convertTimestampWithLocalZone(Column column, Field fieldDefn, Object value) {
-        if (value instanceof Long) {
+        if (value instanceof Number) {
```
Comment (jpechane) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/olr/OpenLogReplicatorValueConverter.java:117]: This does not look safe. You check for `Number` but then you typecaset to `Long`

**Review by jpechane — CHANGES_REQUESTED** [review]
In general it looks good, but there is one instance which I consider to be an unsafe typecasting.

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/olr/OpenLogReplicatorValueConverter.java (hunk 1 above)
Comment (Naros) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/olr/OpenLogReplicatorValueConverter.java:117]: Thanks, I'll take a look next week when back.

**Review by jpechane — APPROVED** [review]
@Naros LGTM, thanks. Feel free to merge it when CI passes.

**Pull request merged**